packages/web-runtime/src/components/Topbar/Notifications.vue

```diff
@@ -291,11 +292,32 @@ export default {
       setAdditionalData()
     }
 
-    onMounted(() => {
+    const setupSSE = async () => {
+      await fetchEventSource('/ocs/v2.php/apps/notifications/api/v1/notifications/sse', {
+        headers: {
+          Authorization: `Bearer ${accessToken.value}`
```

Comment:
Please add 2 more headers: `Accept-Language` and `X-Request-ID`. See here e.g.: https://github.com/owncloud/web/blob/sse-notifications/packages/web-pkg/src/services/client/client.ts#L110